### PR TITLE
Commseries

### PR DESCRIPTION
**Pull request by opotowsky**
An agent commissioning time series directly from the agent entry table

### CODE REVIEW

**Comment by opotowsky** [comment]
anyone care to have a look? @rwcarlsen @Baaaaam @rwcarlsen @gonuke @scopatz

File: cymetric/metrics.py [hunk 1]
```diff
@@ -191,6 +191,27 @@ def decay_heat(series):
...
 
+# Agent Commissioning
+_csdeps = [('AgentEntry', ('SimId', 'EnterTime'), 'Prototype')]
+
+_csschema = [
+    ('SimId', ts.UUID), ('EnterTime', ts.INT), ('Prototype', ts.STRING), 
+    ('Count', ts.INT)
+    ]
+
+@metric(name='CommissioningSeries', depends=_csdeps, schema=_csschema)
```
Comment (rwcarlsen) [inline cymetric/metrics.py]: In Cyclus (kernel repo anyway) we have so far used the convention `Build <--> Decommission`.  That would be my preference here - just to maintain consistency.

Comment (rwcarlsen) [inline cymetric/metrics.py]: ...regarding metric name in case that wasn't clear.

**Comment by rwcarlsen** [comment]
Other than the naming issue, LGTM.

**Comment by opotowsky** [comment]
Updated, thanks @rwcarlsen

**Comment by scopatz** [comment]
Yep, also LGTM. Thanks for putting this in @opotowsky!

**Pull request merged**